go.mod

```diff
@@ -10,44 +10,44 @@ require (
 	github.com/go-kit/kit v0.12.0
 	github.com/go-redis/redis/v8 v8.11.5
 	github.com/golang-jwt/jwt/v4 v4.5.0
-	github.com/google/uuid v1.3.0
+	github.com/google/uuid v1.3.1
 	github.com/lestrrat-go/jwx v0.9.0
-	github.com/optimizely/go-sdk v1.8.4-0.20230911163718-b10e161e39b8
+	github.com/optimizely/go-sdk v1.8.4-0.20240118173841-4adb1affdb93
```

Comment:
Do we need to use the v2 of the SDK now?

Reply:
Yes, I'll do that in another PR after merging that go-sdk PR.